scripts/shellcheck_job.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import argparse
+import pathlib
+import subprocess
+import sys
+import xmltodict
+
+from typing import List, Tuple
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
```

Comment:
This is very nitpicky but since you use `
def process_job(job_obj: dict) -> bool` elsewhere I suggest sticking with either pre-3.9 `from typing import List, Dict, ...` or post-3.9 and use `dict`, `list`, `tuple`, etc as your type hints.  It's not wrong to mix them, but w/in a single file if you use one post-3.9 type hint you might as well do it everwhere else in that file :-)

Reply:
I'm glad you pointed out that inconsistency :)